src/web_interface/templates/show_analysis.html

```diff
@@ -303,7 +303,7 @@ <h5 class="modal-title">Add analysis to file</h5>
                 {% endblock %}
 
                 {# summary of included files #}
-                {%- if firmware.vendor and selected_analysis -%}
+                {%- if selected_analysis -%}
```

Comment:
As discussed we want to check if this can be lazy loaded with a click-of-a-button, so no unnecessary data is transferred while perusing the file tree.

Reply:
What do you think would be the best (most intuitive) text for the button? "Load Summary"? "Generate Summary"? "Show Summary"? Or "Load/Generate/Show Analysis Summary" maybe?

Reply:
I changed to `Load Summary for Included Files` as a short enough and understandable. Changed naming in other places to be consistent.